### ISSUE TITLE
Sell Mons To Server

### ISSUE THREAD

**Issue by Kucing-Kayang** [SmashMC-Development/Suggestions#1012]
**Describe the suggestion**
i Hope server make feature to sell mons to server cause sometimes ultrabeast or legendary with very bad ivs and nature cannot sold in gts since nobody want it, so at least those mons with nobody want it not just going to be trashed. i think it's will be very good even it's not getting some money but maybe some poin system with several reward like random ha or something. So The mons not waste
and i think it's gonna still balance if you limit it to only legandary or ultrabeast since there's lots of breeder exist

**Which servers does this apply to?**
Reshiram/all

**Additional context**
Actually i think poin system it's better then money and not gonna destroy the economy and probably there's feature like you can trade the poin or sell the poin in gts so the economy not stopping and still have something to sell and trade

**Comment by manlybro2002** [comment]
I like the idea of legendary points, makes wz better

**Comment by Firemelon66** [comment]
What u guys have in mind what u could buy with the legend points?

**Comment by WalkAway7** [comment]
Maybe like held items or master balls

**Comment by BrunchBee** [comment]
It'd be interesting to be able to buy things like destiny knots, everstones, ranch upgrades, and held items.

But I definitely like the idea of being able to buy master balls, to help regain a few of the ones used to catch the legendaries in the first place

**Comment by Kucing-Kayang** [comment]
i agree with master ball because that's gonna make us not feel like catching legendary not useless when the ivs so bad
